com.unity.shadergraph/Editor/Data/Util/KeywordUtil.cs

```diff
@@ -79,6 +79,10 @@ public static string ToDeclarationString(this KeywordDescriptor keyword)
                     var enumEntryDefinitions = keyword.entries.Select(x => $"{keyword.referenceName}_{x.referenceName}");
                     string enumEntriesString = string.Join(" ", enumEntryDefinitions);
                     return $"#pragma {definitionString} {enumEntriesString}";
+                case KeywordType.MultiCompile:
+                    var multiCompileEntryDefinitions = keyword.entries.Select(x => $"{x.referenceName}");
+                    string multiCompileEntriesString = string.Join(" ", multiCompileEntryDefinitions);
+                    return $"#pragma {definitionString} {multiCompileEntriesString}";
```

Comment:
this is not necessary. the enum prefix (`keyword.referenceName`) is `RENDER` in this case. the existing enum keyword case already meets this need.